src/ducks/categories/CategoriesHeader.styl

```diff
@@ -45,6 +45,12 @@
     bottom 3.5rem
     left calc(50% - 93px)
 
+.NoAccount_empty
+    position: relative
+    transform: translateZ(0)
+    height: 140px
+    display: flex
```

Comment:
Why is this styling necessary ?

Reply:
To display it correctly in the page

Reply:
if we do not put the style, it display the component in the wrong place

Reply:
See https://docs.cozy.io/cozy-ui/react/#/Content?id=empty

Reply:
OK I think the PR in cozy-ui should be merged so that we have not to do adhoc styling here. Maybe this PR can be mentioned here so that when coming back to the code, we know that the code can be removed ?

Reply:
Yes it's better indeed. So I will remove the style when PR [#1642](https://github.com/cozy/cozy-ui/pull/1642) will be merged